packages/blade/src/components/List/listTokens.ts

```diff
@@ -0,0 +1,152 @@
+import type { CodeProps } from '../Typography';
+import type { ListProps } from './List';
+import type { TypographyPlatforms } from '~tokens/global/typography';
+import type { DotNotationSpacingStringToken } from '~src/_helpers/types';
+
+const listItemPaddingLeft: Record<
+  NonNullable<ListProps['variant'] | 'unorderedWithIcon'>,
+  Record<number, DotNotationSpacingStringToken>
+> = {
+  unordered: {
+    1: 'spacing.2',
+    2: 'spacing.5',
```

Comment:
why do level 2 and level 3 have same padding? shouldn't 3 have more padding 🤔

Reply:
Since level 2 items are rendered inside level 1 items, there is already a padding added by level 1 and this level 2 padding is space in addition to the level 1 space. So if level 1 has 4px padding and we specify 8px padding to level 2 then level 2 will be 4+8=12px from the leftmost point